extensions.json

```
@@ -1444,7 +1444,11 @@
     "repository": "https://github.com/vortizhe/vscode-ruby-erb"
   },
   "vscjava.vscode-gradle": {
-    "repository": "https://github.com/microsoft/vscode-gradle"
+    "repository": "https://github.com/microsoft/vscode-gradle",
+    "location": "extension",
+    "custom": [
+      "cd extension && git clone https://github.com/microsoft/build-server-for-gradle.git && ../gradlew buildJars && cd .. && ./gradlew build && cd extension && vsce package -o extension.vsix"
+    ]
```

Full disclosure, *not an approved reviewer*.

----

I see why it's all `&&`-ed together since each line is [an isolated `exec`](https://github.com/EclipseFdn/publish-extensions/blob/9a5ad339335280eade3448431881f251a2edcd68/scripts/build-extension.js#L119) with an explicit `cwd`... But the `&&` chaining kinda sets off my "ick alarm".

Something like this may be a little easier on the eyes and require less careful reading:

```suggestion
    "custom": [
      "cd extension && git clone https://github.com/microsoft/build-server-for-gradle.git",
      "cd extension && ../gradlew buildJars",
      "./gradlew build",
      "cd extension && vsce package -o extension.vsix"
    ]
```